static/app/components/onboardingWizard/task.tsx

```diff
@@ -65,6 +65,13 @@ function Task(props: Props) {
     recordAnalytics(task, organization, 'clickthrough');
     e.stopPropagation();
 
+    if (task.task === OnboardingTaskKey.SIDEBAR_GUIDE) {
+      localStorage.setItem('sidebarGuide', '1');
```

Comment:
@roggenkemper hmmm...why are we using local storage for this? This means that the  `sidebarGuide` is active?

Reply:
my goal here was to only have the GuideAnchor turn on after they click the tour, else they would get the guide as soon as they enter the sandbox. i thought localStorage would be the best move so that I could track when they entered and then enable the Anchor.

Reply:
@roggenkemper this will use the store that you're making in the future right?